packages/devtools_app/lib/src/utils.dart

```diff
@@ -65,22 +65,27 @@ final NumberFormat nf = NumberFormat.decimalPattern();
 
 String percent2(double d) => '${(d * 100).toStringAsFixed(2)}%';
 
-String prettyPrintBytes(num bytes, {bool includeUnit = false}) {
+String prettyPrintBytes(
+  num bytes, {
+  int fractionDigits = 0,
```

Comment:
we should probably have two parameters `kbFractionDigits` (defaults to 0), and `mbFractionDigits` (defaults to 1 to match `printMb`).

It feels natural to need different granularity defaults at different orders of magnitude.

Reply:
Makes sense! Done